### PR TITLE
Update System Requirements

### PR DESCRIPTION
**Pull request by andrewdavidwong**

### CODE REVIEW

**Comment by marmarek** [comment]
Extracted from CI log:
```
- ./_site/community/index.html
  *  linking to internal hash #qubes-release-4x that does not exist (line 722)
     <a href="/doc/system-requirements/#qubes-release-4x">4.x System Requirements</a>
```

File: user/hardware/system-requirements.md [hunk 1]
```diff
@@ -19,69 +19,51 @@ redirect_from:
...
+ - See the [Certified Hardware] page.
+ - See the [Hardware Testing] page.
 
-## Important Notes ##
+## Important Notes
 
- * **Installing Qubes in a virtual machine is not recommended, as it uses its own bare-metal hypervisor (Xen).**
- * Qubes **can** be installed on systems which do not meet the recommended requirements.
+ - **Installing Qubes in a virtual machine is not recommended, as it uses its own bare-metal hypervisor (Xen).**
+ - Qubes **can** be installed on systems which do not meet the recommended requirements.
```
Comment (marmarek) [inline user/hardware/system-requirements.md:55]: See https://github.com/QubesOS/qubes-doc/pull/1085#discussion_r524300888 - maybe we should include similar note here too?

Comment (andrewdavidwong) [reply user/hardware/system-requirements.md:55]: Actually, that comment is what prompted me to update this page.

I don't think it's necessary to include that note. That information is only of value to advanced technical users who wish to install 4.0 but not 4.1, which is a rapidly shrinking group of people, possibly soon to round to zero. If we document that somewhere, it should be on a page for advanced users, or simply here on GitHub, not in a place where everyone (including non-technical users) will have to read it and be confused by it.

Comment (marmarek) [reply user/hardware/system-requirements.md:55]: Yes, makes sense.

**Comment by andrewdavidwong** [comment]
> Extracted from CI log:
> 
> ```
> - ./_site/community/index.html
>   *  linking to internal hash #qubes-release-4x that does not exist (line 722)
>      <a href="/doc/system-requirements/#qubes-release-4x">4.x System Requirements</a>
> ```

Thanks, but there seems to be a bug in this new CI (already mentioned to @fepitre on an unrelated PR). This log says `./_site/community/index.html`, but in fact, those old slugs are on different pages:

```
$ grep -r '#qubes-release-4x'
user/hardware/certified-hardware.md:[system requirements for Qubes 4]: /doc/system-requirements/#qubes-release-4x
introduction/faq.md:[4.x System Requirements]: /doc/system-requirements/#qubes-release-4x 
```

It would be handy if we could get back the old functionality that correctly identifies the pages, but of course I can simply grep for them in the meantime. Fixing these now.

**Comment by andrewdavidwong** [comment]
While fixing the old slug on the FAQ page, I found more old content related to system requirements to update and improve.

File: introduction/faq.md [hunk 2]
```diff
@@ -367,46 +367,43 @@ This also means that it is possible to update the software for several qubes sim
...
-### Can I install Qubes OS on a system without VT-x?
+### Why is VT-x/AMD-V important?
 
-Yes, for releases 3.2.1 and below. 
-Xen doesn't use VT-x (or AMD-v) for PV guest virtualization. 
+Xen doesn't use VT-x/AMD-V for PV guest virtualization. 
 (It uses ring0/3 separation instead.) 
-However, without VT-x, you won't be able to use fully virtualized VMs (e.g., Windows-based qubes), which were introduced in Qubes 2. 
-In addition, if your system lacks VT-x, then it also lacks VT-d. (See next question.)
+However, without VT-x/AMD-V, you won't be able to use fully virtualized VMs (e.g., Windows-based qubes).
```
Comment (marmarek) [inline introduction/faq.md:383]: In 4.x we use HVM/PVH (which requires VT-x/AMD-V) for Linux VMs too, so this needs an update. Basically without VT-x/AMD-V no VM will start in default configuration.

**Review by marmarek — CHANGES_REQUESTED** [review]

File: introduction/faq.md (hunk 2 above)
Comment (andrewdavidwong) [reply introduction/faq.md]: Thanks, updated.

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of 233e45ba71c434c6ff17fc438020ae34f956409d
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAl+62SoACgkQ8yiUvpaE
k4o4RRAApPpdhizvI0Yzggeg8x8YAvSk/q0qhl8XjBQTI52OlJV+YpM96/Yf5hUP
aZ8cs8dsEDiO4Fjg1YwXKBAFLKCQJwRd0RR6WiNFAIWqkYCt4MebIKGK4Y6OBJd9
J+3i6Neh7OqGMSZF30KBhBIIPdtf2EVECEjNgMveC81ywAI7mdSszws1py/H+TNq
4cMhffuKR6FgWOTZg1pJ3NsnSeTjtAwNtJsiQmfe0DVmX/Gg2K9Ehj0zHchwh+TB
4GPdHUYzkhNduGV1A8wgq5KuvpMw2e8CrgKiKCJ/9uvrlKHSbSgwS0CcXlU9x3O9
ZsL1z6Dx0TW3gRUdpaEbNTQUn/WyCbGtgFw1mFnGkKYF1qqgx1hDXlIvY0TUNfd/
SddCx+Pqept2fnkm/E7LtGlNx+8BqgmGoGiwD4dydD1byzgGdMtm17lnMUXSfvqS
yxpVbYiyv4aUkeYyX2Aih5mGrvbxijMmDDc9vwCf8JYlPMapYlKyg8+WCzbg02fS
XzaBFqjhM5ojEnUPyRstVp/VTEUnFujlmEIi06M9sGTDS+zpuP59vFA1vC+ivmoO
C7GnvvlgSnIO9GA5mlu17bCeM8EGN4hvL4crgnrA8N3gzmHe27fjM/SJyylrXKO9
larIdhjHd9CVE9W2G2DANDQD5M9q2c6PlHBLMTpeUjMX3gDkOLs=
=n7Tx
-----END PGP SIGNATURE-----
```

**Comment by marmarek** [comment]
> Thanks, but there seems to be a bug in this new CI (already mentioned to @fepitre on an unrelated PR). This log says `./_site/community/index.html`, but in fact, those old slugs are on different pages:
> 
> It would be handy if we could get back the old functionality that correctly identifies the pages, but of course I can simply grep for them in the meantime. Fixing these now.

I've identified it an issue in the new version of html-proofer that coincided with out CI migration: https://github.com/gjtorikian/html-proofer/issues/602
I'll force the old version for now.

**Pull request merged**